docs/get-started/quickstarts/control-motor.md

```diff
@@ -21,30 +21,41 @@ Follow these steps to control your motor:
 
 {{< expand "Step 1: Create a machine" >}}
 
-Go to the Viam app and [add a new machine](/cloud/machines/#add-a-new-machine).
-
-![The 'First Location' page on the Viam app with a new machine name in the New machine field and the Add machine button next to the field highlighted.](/fleet/app-usage/create-machine.png)
```

Comment:
Sorry didn't notice and comment on this before--we actually got positive feedback on it from a user in another quickstart--I think it allows people to quickly see what they need to do without having to click through. I see that we want to have all the machine names agree, so maybe take a quick screenshot with the correct machine name to replace this one for this quickstart? Shouldn't take long but if not in this PR, mind making a ticket to add one back in later?